### DISCUSSION TITLE
Minimum width and height on a desktop Blazor application

### DISCUSSION THREAD

**Discussion by ccarv** [dotnet/maui#10017] [Q&A] [answered]
I would like to define constraints for minimum width and height when building towards desktop platforms. If I resize my application to the point it reaches a width defined in my CSS media queries, it will reorient the layout to support a mobile view. I do not want this on a desktop application.

The research I've done has led me to solutions addressing a "fixed" width and height. I haven't been able to find a workable solution that allows for resizing of the window and enforcing a minimum width and height.

**Comment by xtuzy** [comment, marked as answer]
On Windows you can use https://github.com/dotMorten/WinUIEx

**Reply by ccarv** [reply]
This worked. Thank you for the recommendation.

[Here is the documentation](https://dotmorten.github.io/WinUIEx/concepts/Maui.html) for MAUI.
